docs/web3.eth.rst

```diff
@@ -282,7 +282,7 @@ The following methods are available on the ``web3.eth`` namespace.
           'gasLimit': '0x2fefd8',
           'gasUsed': '0x0',
           'hash': '0xc78c35720d930f9ef34b4e6fb9d02ffec936f9b02a8f0fa858456e4afd4d5614',
-          'logsBloom':'0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,
+          'logsBloom':'0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
Nice catch! How do you feel about turning these into doctests so that we catch errors like this in the future? If it's too much for this PR, I'll add a new issue. I think it is a good idea to start moving that direction for all of our docs.

Reply:
Agreed, I'm all for doctest and autodoc, but I think it's a somewhat hefty task and deserves it's own issue / pr(s).